src/main/java/com/sarahhand/fractals/juliaset/JuliaConfig.java

```diff
@@ -114,13 +127,14 @@ private JuliaConfig() {}
 	 * @param palette
 	 * @param colorScheme
```

Comment:
Update the Javadoc to include the new parameter and a description of what they are.